### PR TITLE
LC CAC announcement of new lesson 

### PR DESCRIPTION
**Pull request by jt14den**
This is a blog entry announce a new data management planning lesson for librarians and information professionals.

### CODE REVIEW

**Review by tobyhodges — APPROVED** [review]
Loos good to me, thanks all

File: _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Marla Hertz is an Associate Professor and the Research Data Management Librarian (RDM) at the University of Alabama at Birmingham (UAB). UAB is a public research-intensive university with roughly 21K students and over $700 million dollars of extramural research awards last year. Marla has a PhD in microbiology and 15+ years of research experience. Marla transitioned into RDM librarianship 2 years ago to support the data management and sharing needs of UAB researchers.
+
+Daria Orlowska is a data librarian and assistant professor at Western Michigan University, a public doctorate-granting university with ~17,000 students with high research activity by the Great Lakes in the United States. Daria has a bachelor’s degree in psychology and a master’s degree in library and information science and experience coordinating federally-funded health psychology research full-time for several years. Daria has been in my role for 4.5 years. As part of her role, Daria provides data education and services to students, faculty, and staff within the university community.
```
Comment (tobyhodges) [inline _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md:46]:
```suggestion
Daria Orlowska is a data librarian and assistant professor at Western Michigan University, a public doctorate-granting university with ~17,000 students with high research activity by the Great Lakes in the United States. Daria has a bachelor’s degree in psychology and a master’s degree in library and information science and experience coordinating federally-funded health psychology research full-time for several years. Daria has been in her role for 4.5 years. As part of her role, Daria provides data education and services to students, faculty, and staff within the university community. 
```

Comment (OscarSiba) [reply _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md:46]: Thank you @tobyhodges!

File: _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: "Library Carpentry Curriculum Committee Approves Data Management Plan Lesson"
+date: 2024-05-27
+time: "09:00:00"
+tags: ["Curriculum", "Library Carpentry", "Maintainers", "Curriculum Development", "IMLS"]
+
+---
+
+### Introduction
+
+We are excited to announce that the Library Carpentry Curriculum Advisory Committee (LC-CAC) has approved a new lesson Data Management Plan Course for Librarians [http://librarycarpentry.org/lc-dmp101/](http://librarycarpentry.org/lc-dmp101/) for inclusion in the [Library Carpentry curriculum](https://librarycarpentry.org/lessons/). This course was developed as part of an Institute for Museum and Library Services (IMLS) grant to UCLA to develop open science lessons for Library Carpentry.
```
Comment (OscarSiba) [inline _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md:14]:
```suggestion
We are excited to announce that the Library Carpentry Curriculum Advisory Committee (LC-CAC) has approved a new lesson [Data Management Plan Course for Librarians](http://librarycarpentry.org/lc-dmp101/) for inclusion in the [Library Carpentry curriculum](https://librarycarpentry.org/lessons/). This course was developed as part of an Institute for Museum and Library Services (IMLS) grant to UCLA to develop open science lessons for Library Carpentry.
```

File: _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. **Data Management Plan Overview**. This episode lays out the components of an ideal DMP and covers important background information.
+
+2. **DMP Resources**. This episode includes resources for librarians to use when assisting DMPs, including how to find funder requirements, repositories, data standards, and example DMPs.
+
+3. **Supporting Researchers**. This episode focuses on how to conduct an effective data interview.
+
+4. **DMPTool and Common DMP Issues**. This episode includes a demo of the DMPTool application and provides solutions for working with researchers on common DMP issues.
+
+5. **Data Management Plan Services**. This episode provides strategies for starting a data management plan service in the library.
```
Comment (OscarSiba) [inline _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md:30]:
```suggestion
5. **Data Management Plan Services**. This episode provides strategies for starting a DMP service in the library.
```

File: _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Why a DMP course for librarians?
+
+By the end of this course, librarians should be equipped to offer basic DMP review services to researchers.
```
Comment (OscarSiba) [inline _posts/2024/05/2024-05-27-library-carpentry-dmp-lesson-approved.md:34]:
```suggestion

```

**Comment by OscarSiba** [comment]
Hey @jt14den have you had time to look at some of the suggested changes to the post? We can then schedule to release later this week or early next week.

**Comment by jt14den** [comment]
looks good @OscarSiba, good to go.

**Comment by jt14den** [comment]
@OscarSiba any sense on when this will go live?

**Comment by OscarSiba** [comment]
> @OscarSiba any sense on when this will go live?

We're releasing it today @jt14den. :-)

**Pull request merged**